src/components/register/register.jsx

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable react/no-children-prop */
+import React, { useState } from 'react';
+import { instanceOf } from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import {
+  FormControl,
+  FormLabel,
+  InputGroup,
+  Input,
+  Heading,
+  InputRightAddon,
+  Button,
+  Flex,
+  Link,
+  Spacer,
+  VStack,
+  Center,
+  Alert,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Alert' is defined but never used.